lib/pages/main.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import '../widgets/focus_box.dart';
+import '../widgets/like_button.dart';
+import '../widgets/movie_header.dart';
+import '../widgets/movie_overview.dart';
+import '../widgets/movie_title.dart';
+import '../widgets/vote_average.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  messagePrint() {
+    print('Function Called');
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        useMaterial3: true,
+        primaryColor: Colors.deepOrange.shade900,
+        colorScheme: ColorScheme.fromSeed(
+          seedColor: Colors.deepOrange.shade900,
+          background: Colors.deepOrange.shade900,
+        ),
+      ),
+      home: Builder(
```

Comment:
I'd re-arrange this as the following:

- `lib/main.dart`: with the runApp method and a basic `MyApp` class.
- `lib/pages/movie_details.dart`: with a new widget with the `Scaffold` and so.

So that you have your entry point (main.dart) more accessible!